src/__test__/redux/actions/cellSets/reorderCellSet.test.js

```diff
@@ -0,0 +1,76 @@
+import configureStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import fetchMock, { enableFetchMocks } from 'jest-fetch-mock';
+import waitForActions from 'redux-mock-store-await-actions';
+import { waitFor } from '@testing-library/react';
+
+import reorderCellSet from 'redux/actions/cellSets/reorderCellSet';
+import { CELL_SETS_REORDER } from 'redux/actionTypes/cellSets';
+import pushNotificationMessage from 'utils/pushNotificationMessage';
+
+import initialState from 'redux/reducers/cellSets/initialState';
+
+import '__test__/test-utils/setupTests';
+
+enableFetchMocks();
+const mockStore = configureStore([thunk]);
```

Comment:
Can we use the real store instead of a mock store?

Reply:
For action creators shouldn't we focus on the actions they dispatch? Why do we need the real store?

Reply:
Hmm... ok